OpenSim/Simulation/Wrap/WrapSphereObst.h

```diff
@@ -42,48 +43,36 @@ class WrapResult;
  */
 class OSIMSIMULATION_API WrapSphereObst : public WrapObject {
 OpenSim_DECLARE_CONCRETE_OBJECT(WrapSphereObst, WrapObject);
-
-//=============================================================================
-// DATA
-//=============================================================================
-private:
-    PropertyDbl _radiusProp;
-    double& _radius;
-
-    PropertyDbl _lengthProp;
-    double& _length;
+public:
+//==============================================================================
+// PROPERTIES
+//==============================================================================
+    OpenSim_DECLARE_PROPERTY(radius, double, "The radius of the cylinder.");
```

Comment:
Intentional mismatch between class name (sphere) and "cylinder" here?

Reply:
My bad... sry